content/en/hosting/analytics/environment-variables.md

```diff
@@ -26,11 +24,15 @@ All the variables in the `.env` file:
 | `POSTGRES_PORT`           | `5432`                                                | PostgreSQL port  |
 | `CHT_PIPELINE_BRANCH_URL` | `"https://github.com/medic/cht-pipeline.git#main"`    | cht-pipeline branch containing the dbt models                                                                                            |
-| `DATAEMON_INTERVAL`       | `5`                                                   | Interval (in minutes) for looking for new changes in the CouchDB data                                                                      |
+| `DATAEMON_INTERVAL`       | `5`                                                   | Interval (in seconds) for looking for new changes in the CouchDB data                                                                      |
 | `COUCHDB_USER`            | `medic`                                               | Username of the CouchDB instance                                                                                             |
 | `COUCHDB_PASSWORD`        | `password`                                            | Password of the CouchDB instance                                                                                              |
 | `COUCHDB_DBS`             | `"medic"`                                             | Comma separated list of databases to sync e.g `"medic, medic_sentinel"`                                                                     |
 | `COUCHDB_HOST`            | `couchdb`                                             | Host of the CouchDB instance                                                                                                 |
 | `COUCHDB_PORT`            | `5984`                                                | Port of the CouchDB instance                                                                                                 |
-| `COUCHDB_SECURE`          | `false`                                               | Is connection to CouchDB instance secure?                                                                                                  |
+| `COUCHDB_SECURE`          | `false`                                               | Does the connection to CouchDB use HTTPS? |
+| `DBT_THREAD_COUNT`          | 1                                               | Number of threads per DBT process |
+| `DBT_BATCH_SIZE`          | 0                                               | Batch size for batched incremental runs |
+| `DBT_LOCAL_PATH`          |                                                | When running DBT locally for development, the path to the models directory on the host |
+| `DBT_SELECTOR`          | ''                                               | If using separate DBT processes, the select condition to select a subset of the models for a single dbt process. |
```

Comment:
Please add  either happy path values or commented out values for each of these in the [docker .env file](https://github.com/medic/cht-sync/blob/main/env.template) - thanks! Oh - maybe the `helm` [values file](https://github.com/medic/cht-sync/blob/main/deploy/cht_sync/values.yaml.template) too?

Reply:
I'm now using the `v2` [branch](https://github.com/medic/cht-sync/pull/202/files) on sync and see you added default values - resolving!